pocs/mount/simulator.py

```diff
@@ -91,6 +93,20 @@ def move_direction(self, direction='north', seconds=1.0):
         self.logger.debug("Mount simulator moving {} for {} seconds".format(direction, seconds))
         time.sleep(seconds)
 
+    def get_ms_offset(self, offset, axis='ra'):
+        """ Fake offset in milliseconds
+
+        Args:
+            offset (astropy.units.Angle): Offset in arcseconds
+
+        Returns:
+             astropy.units.Second: Offset in milliseconds at current speed
```

Comment:
How does the type (Second) match with the comment saying it is milliseconds?

Reply:
Actually this should be a`Quantity` anyway. I'll get it changed.